fastlane-plugin-revenuecat.gemspec

```diff
@@ -0,0 +1,36 @@
+lib = File.expand_path("lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'fastlane/plugin/revenuecat/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'fastlane-plugin-revenuecat'
+  spec.version       = Fastlane::Revenuecat::VERSION
+  spec.author        = 'Toni Rico'
+  spec.email         = 'toni.rico.diez@revenuecat.com'
+
+  spec.summary       = 'A plugin including commonly used automation logic for RevenueCat SDKs.'
+  spec.homepage      = "https://github.com/RevenueCat/fastlane-plugin-revenuecat"
+  spec.license       = "MIT"
+
+  spec.files         = Dir["lib/**/*"] + %w(README.md LICENSE)
+  spec.require_paths = ['lib']
+
+  spec.required_ruby_version = '>= 2.5'
+
+  # Don't add a dependency to fastlane or fastlane_re
+  # since this would cause a circular dependency
+
+  # spec.add_dependency 'your-dependency', '~> 1.0.0'
+
+  spec.add_development_dependency('bundler')
+  spec.add_development_dependency('fastlane', '2.191.0')
+  spec.add_development_dependency('pry')
+  spec.add_development_dependency('rake')
+  spec.add_development_dependency('rspec')
+  spec.add_development_dependency('rspec_junit_formatter')
+  spec.add_development_dependency('rubocop', '1.31.1')
```

Comment:
I updated this version to the latest rubocop version to solve some issues I was getting